integrations/api_repo_test.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	"code.gitea.io/gitea/models"
 
+	api "code.gitea.io/sdk/gitea"
```

Comment:
Should be next to other `code.gitea.io` imports (see [the styleguide](https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#styleguide))